src/main/features/claim/views/check-and-send.njk

```diff
@@ -107,7 +107,7 @@
     {{ tableEnd() }}
 
     {{ tableStart('claimDetails', 'Claim details') }}
-    {{ row('Why you’re owed the money:', draftClaim.reason.reason, paths.reasonPage.uri ) }}
+    {{ row('Why you’re owed the money:', draftClaim.reason.reason, paths.reasonPage.uri, bold = true ) }}
```

Comment:
I wonder if bold shouldn't become a default value in the future ;)

Reply:
true. Will have to look into that at some point